installer/manifests/keptn/charts/control-plane/templates/rbac.yaml

```diff
@@ -252,46 +252,6 @@ subjects:
   - kind: ServiceAccount
     name: keptn-lighthouse-service
 
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: keptn-api-service-manage-bridge-secret
-  labels:
-    app.kubernetes.io/name: keptn-api-service-manage-bridge-secret
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/part-of: keptn-{{ .Release.Namespace }}
-    app.kubernetes.io/component: {{ include "control-plane.name" . }}
-    helm.sh/chart: {{ include "control-plane.chart" . }}
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: keptn-manage-bridge-secret
```

Comment:
You did not delete the role keptn-manage-bridge-secret and keptn-delete-bridge-pod. Is this role used somewhere else? If no, please also delete the role.

Reply:
removed